### PR TITLE
Update Splunk Add-on Article

### PR DESCRIPTION
**Pull request by MichaelYochpaz**
## Status
In Progress

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-9536)

## Description
Update the [Splunk Add-on article](https://xsoar.pan.dev/docs/reference/articles/splunk-add-on)

### CODE REVIEW

**Review by MosheEichler — APPROVED** [review]

**Review by eyalpalo — COMMENTED** [review]
Very nice!! see my comments, and lets move it to doc review.

File: content-repo/extra-docs/articles/splunk-add-on.md [hunk 1]
```diff
@@ -5,171 +5,197 @@ description: Splunk add-on for Cortex XSOAR. This add-on enables you to push inc
...
   ```
-    12-28-2021 13:45:13.807 -0800 ERROR sendmodalert - action=create_xsoar_incident STDERR - NameError: name 'basestring' is not defined
-    12-28-2021 13:45:13.807 -0800 ERROR sendmodalert - action=create_xsoar_incident STDERR - if isinstance(self.sid, basestring) and 'scheduler' in self.sid:
-    12-28-2021 13:45:13.807 -0800 ERROR sendmodalert - action=create_xsoar_incident STDERR - File "/opt/splunk/etc/apps/Splunk_SA_CIM/lib/cim_actions.py", line 157, in init
+  12-28-2021 13:45:13.807 -0800 ERROR sendmodalert - action=create_xsoar_incident STDERR - NameError: name 'basestring' is not defined
+  12-28-2021 13:45:13.807 -0800 ERROR sendmodalert - action=create_xsoar_incident STDERR - if isinstance(self.sid, basestring) and 'scheduler' in self.sid:
+  12-28-2021 13:45:13.807 -0800 ERROR sendmodalert - action=create_xsoar_incident STDERR - File "/opt/splunk/etc/apps/Splunk_SA_CIM/lib/cim_actions.py", line 157, in init
   ```
+
+### Logs
```
[the remark this answers is not in the document]
Comment (MichaelYochpaz) [reply content-repo/extra-docs/articles/splunk-add-on.md]: You meant to the top of the "Troubleshooting" section like I did [here](https://github.com/demisto/content-docs/pull/1470/commits/94009da55c226638d4034a39f20bd1603b68f5cf), or as a separate top-level header (`## Logs`) above "Troubleshooting"?

File: content-repo/extra-docs/articles/splunk-add-on.md [hunk 2]
```diff
@@ -5,171 +5,197 @@ description: Splunk add-on for Cortex XSOAR. This add-on enables you to push inc
...
-   curl -kv -H "Authorization:<API_KEY>" https://<XSOAR_SERVER>:<PORT>/user
-   wget --no-check-certificate --header="Authorization: <API_KEY>" -O - https://<XSOAR_SERVER>:<PORT>/user
+4. Test the API key generated in Cortex XSOAR.
+   ```bash
+   curl -kv -H "Authorization:<API_KEY>" https://<XSOAR_SERVER>/user
+   wget --no-check-certificate --header="Authorization: <API_KEY>" -O - https://<XSOAR_SERVER>/user
    ```
    
 #### Connection Timeout Errors
+Connection timeout errors can be identified in the Splunk logs, and will look as follows:
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/splunk-add-on.md:161]:
```suggestion
Connection timeout errors can be identified in the Splunk logs, and will appear as follows:
```

File: content-repo/extra-docs/articles/splunk-add-on.md [hunk 3]
```diff
@@ -5,171 +5,197 @@ description: Splunk add-on for Cortex XSOAR. This add-on enables you to push inc
...
 1. Locate the `SavedSplunker` log facility.
 2. Increase its verbosity to *ERROR* level.
-3. If the logs include the following:
-    ```WARN SavedSplunker - Reached maximum amount of time allowed to spend in per-result alerts for savedsearch_id="SAVEDSEARCH_ID"```
-    Increase the *max_per_result_alerts_time* value in the `limits.conf` file.
+3. If the logs include the following:  
+   ```
+   WARN SavedSplunker - Reached maximum amount of time allowed to spend in per-result alerts for savedsearch_id="SAVEDSEARCH_ID"
+   ```  
+   Increase the ***max_per_result_alerts_time*** value in the `limits.conf` file.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/splunk-add-on.md:172]:
```suggestion
   Increase the *max_per_result_alerts_time* value in the `limits.conf` file.
```

File: content-repo/extra-docs/articles/splunk-add-on.md [hunk 4]
```diff
@@ -5,171 +5,197 @@ description: Splunk add-on for Cortex XSOAR. This add-on enables you to push inc
...
 
-#### Incident Created with Incorrect Fields
-If incidents being created in XSOAR but with incorrect fields, refer to the [Connectivity Test](#connectivity-test---create-a-custom-alert-action-from-saved-searches) section for an understanding of the fields which are associated with create incident actions.
+#### Incidents Created in Cortex XSOAR with Incorrect Fields
+If incidents are being created in Cortex XSOAR with incorrect fields, ensure that the ***Create XSOAR Incident*** trigger action for the alert is configured properly.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/splunk-add-on.md:175]:
```suggestion
If incidents are being created in Cortex XSOAR with incorrect fields, ensure that the **Create XSOAR Incident** trigger action for the alert is configured properly. 
```

File: content-repo/extra-docs/articles/splunk-add-on.md [hunk 5]
```diff
@@ -5,171 +5,197 @@ description: Splunk add-on for Cortex XSOAR. This add-on enables you to push inc
...
+If incidents are being created in Cortex XSOAR with incorrect fields, ensure that the ***Create XSOAR Incident*** trigger action for the alert is configured properly. 
+For more information about the fields mapping, see the [Alert Configuration](#alert-configuration) section a
 
-#### Too Many Incidents Created in XSOAR
+#### Too Many Incidents Are Created in Cortex XSOAR
 Splunk has its own trigger conditions using the number of incidents which get pushed to XSOAR controlled.
 
-Refer to [Splunk Documentation](https://docs.splunk.com/Documentation/Splunk/latest/Alert/AlertTriggerConditions) to understand Splunk Trigger Conditions to limit the incidents. 
-Also, it is possible to [throttle alerts](https://docs.splunk.com/Documentation/SplunkCloud/latest/Alert/ThrottleAlerts) in Splunk after the saved search returned positive results, in order to suppress more incident creation.
+See Splunk's *[Configure alert trigger conditions](https://docs.splunk.com/Documentation/Splunk/latest/Alert/AlertTriggerConditions)* documentation to understand Splunk Trigger Conditions to limit the incidents.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/splunk-add-on.md:181]:
```suggestion
See Splunk's [Configure alert trigger conditions](https://docs.splunk.com/Documentation/Splunk/latest/Alert/AlertTriggerConditions) documentation to understand Splunk Trigger Conditions to limit the incidents.  
```

File: content-repo/extra-docs/articles/splunk-add-on.md [hunk 6]
```diff
@@ -5,171 +5,197 @@ description: Splunk add-on for Cortex XSOAR. This add-on enables you to push inc
...
 
 #### Playbooks not getting invoked on Cortex XSOAR
-Cortex XSOAR allows users to set up different types of incidents and the user can associate different playbooks the each type of incident. If after pushing incidents from Splunk, the appropriate playbooks are not being invoked, you should check the following:
-1. Check if the type of incident which is getting pushed from Splunk has a playbook associated with the Type in XSOAR.
-
-2. Check if there are no case differences in the Type created in XSOAR and the type which is being pushed along with incident information in Splunk.
+Cortex XSOAR allows users to set up different types of incidents, and the user can associate different playbooks for each type of incident. If after pushing incidents from Splunk, the appropriate playbooks are not being invoked, you should check the following:
+1. Check if the type of incident which is getting pushed from Splunk has a playbook associated with the Type in Cortex XSOAR.
+2. Check if there are no case differences in the Type created in Cortex XSOAR and the type which is being pushed along with incident information in Splunk.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/splunk-add-on.md:187]:
```suggestion
1. Check if the type of incident which is getting pushed from Splunk has a playbook associated with the type in Cortex XSOAR.
2. Check if there are no case differences in the type created in Cortex XSOAR and the type which is being pushed along with incident information in Splunk.
```

File: content-repo/extra-docs/articles/splunk-add-on.md [hunk 7]
```diff
@@ -5,171 +5,197 @@ description: Splunk add-on for Cortex XSOAR. This add-on enables you to push inc
...
 
 #### SSL Certificates Issues
-* If you don’t use a certificate, make sure the **"Validate SSL"** checkbox is not selected.
-
-* If the you have a self-signed certificate, you must add it to the Splunk server first, and then set the path to it in the Add-on configuration page.
-
+* If you don’t use a certificate, make sure the ***Validate SSL*** checkbox is not selected.
```
Comment (ShirleyDenkberg) [inline content-repo/extra-docs/articles/splunk-add-on.md:190]:
```suggestion
* If you don’t use a certificate, make sure the **Validate SSL** checkbox is not selected.
```

File: content-repo/extra-docs/articles/splunk-add-on.md [hunk 8]
```diff
@@ -5,171 +5,197 @@ description: Splunk add-on for Cortex XSOAR. This add-on enables you to push inc
...
+   ![splunk-add-on-configuration-add-xsoar-instance.png](../../../docs/doc_imgs/reference/SplunkAddOn/splunk-add-on-configuration-add-xsoar-instance.png)
+4. Configure a Cortex XSOAR instance:
+   - Configuration for Cortex XSOAR 6 instances:
+     1. Enter a unique instance name on the ***XSOAR Instance name*** field.
+     2. Enter your Cortex XSOAR server URL (including port if needed) on the ***XSOAR Server URL*** field.
+     3. Enter your API key on the ***API Key*** field.  
+        An API key can be generated on XSOAR by navigating to **Settings** → **Integrations** → **API Keys**.
+     4. Click ***Add***. 
+   - Configuration for Cortex XSOAR 8 or Cortex XSIAM instances:
+     1. Enter a unique instance name on the ***XSOAR Instance name*** field.
```
[the remark this answers is not in the document]
Comment (MichaelYochpaz) [reply content-repo/extra-docs/articles/splunk-add-on.md:39]: No, this is a field on a form with a constant name (will always be "XSOAR Instance name")

**Review by bziser — APPROVED** [review]

**Pull request merged**